src/components/SearchInput/SearchInput.spec.js

```diff
@@ -32,6 +33,15 @@ describe('SearchInput', () => {
     expect(actual).toMatchSnapshot();
   });
 
+  it('should display a clear icon when not empty and an onClear callback is provided', () => {
+    const onClear = jest.fn(identity);
+
+    const actual = create(
+      <SearchInput value="search value" onClear={onClear} />
```

Comment:
In addition to (or instead of) the snapshot, would it make sense to use a data-selector to check that the clear icon is there?

Snapshots frequently get updated (for instance when we change theme values) and we might break the test by mistake. WDYT?

Reply:
Sure, I'll update it. I was just following the structure of the other tests, but your comment makes total sense. Thanks!

Reply:
On a second thought when I searched for `data-selector` attribute usage, there were no results. So, I'll leave it as it is for now :)